c7n/filters/core.py

```diff
@@ -1018,3 +1020,123 @@ def reorder(self, items, key=None):
             return items[::-1]
         else:
             return sorted(items, key=key, reverse=(self.order == 'desc'))
+
+
+class FakeModel:
+    id = 'c7n:_id'
+
+
+class FakeFilterRegistry(FilterRegistry):
+
+    def __init__(self, *args, **kw):
+        super(FilterRegistry, self).__init__(*args, **kw)
```

Comment:
you can omit the base class and self params to super, that's python 2 syntax that's extraneous with py3. https://docs.python.org/3/library/functions.html#super